Example/Example/Model/Model.swift

```diff
@@ -1,7 +1,7 @@
 import Faro
 
 /// Example model
-class Model: Parseable {
+class Model: Deserializable {
```

Comment:
Ik heb dat gedaan omdat de naamgeving belangrijk is om te vatten waar het over gaat. Ik ben zelf ook vrij veel tijd kwijt geweest omdat ik dacht dat er in [Kakapo](https://github.com/devlucky/Kakapo) een *Serializable* instance zat die in 2 richtingen werkt. 

Background kan je vinden op [wikipedia](https://en.wikipedia.org/wiki/Serialization) maar serialization zijn computer termen die gangbaar zijn. `Parseable` is dat niet.

*Parsen* = Parsing or syntactic analysis is the process of analysing a string of symbols, either in natural language or in computer languages, conforming to the rules of a formal grammar. The term parsing comes from Latin pars (orationis), meaning part (of speech).

*Serialization* = In computer science, in the context of data storage, serialization is the process of translating data structures or object state into a format that can be stored (for example, in a file or memory buffer, or transmitted across a network connection link) and reconstructed later in the same or another computer environment.

*Deserialization* = (wordt het best vanuit het andere uitgelegd) Serialization is the process of converting an object into a stream of bytes in order to store the object or transmit it to memory, a database, or a file. Its main purpose is to save the state of an object in order to be able to recreate it when needed. The reverse process is called deserialisation.

# Voorstel

Ik stel voor dat we het veranderen en dicht bij de naamgeving van Apple `JSONSerialization` blijven.

Dus wat willen we doen:

1. JSON Dictionary -> Model
2. Model -> JSON Dictionary

Aan die JSON Dictionary geraken we door `JSONSerialization` te gebruiken. 

1. Data -> JSON Dictionary  = `jsonObject(with:options:)`
2. JSON Dictionary -> Data = `writeJSONObject(from:to)

*Dus* 🛢
`Deserializable` =>`DictionaryWritable` 
`Serialisable` => `DictionaryReadable`
`CustomSerializable` => `CustomDictionaryReadable`

Maar ik blijf wel een voorstander van het opsplitsen. Doel van Faro is toch dat we zo weinig methodes maken waar je Nil op moet retourneren. Dat zou je wel moeten doen als je de opsplitsing niet maakt tussen `Deserializable` & `Serialisable`. 

Maar terug gaan naar `Parseable` kan voor mij. Ik zal bovenstaande naamgeving aanpassen en zien wat je vind.